mava/networks.py

```diff
@@ -159,6 +163,54 @@ def __call__(self, obs_embedding: chex.Array, observation: Observation) -> tfd.I
         )
 
 
+class RecQNetwork(nn.Module):
+    """Recurrent Q-Network."""
+
+    num_actions: int
+    hidden_size: int
```

Comment:
I updated things to look as required  - @sash-a @RuanJohn does this look okay to you? The call function feels to me like an action head, but I am not sure how to separate it from the network in a way that I know works, since we need the action head to call get_q_values

Reply:
I believe we decided to leave the addition of a continuous action head to another day. For now, resolved!